hotspot/test/runtime/containers/docker/DockerBasicTest.java

```diff
@@ -75,5 +76,18 @@ private static void testHelloDocker() throws Exception {
             .shouldHaveExitValue(0)
             .shouldContain("Hello Docker");
     }
+
+
+    private static void testJavaVersionWithCgMounts() throws Exception {
+        DockerRunOptions opts =
+            new DockerRunOptions(imageNameAndTag, "/jdk/bin/java", "-version")
+            .addDockerOpts("-v", "/sys/fs/cgroup:/cgroups-in:ro");
+
+        // Duplicated cgroup mounts should be handled by the container detection
+        // code and should not cause any error/warning output.
+        DockerTestUtils.dockerRunJava(opts)
+            .shouldHaveExitValue(0)
+            .shouldNotMatch("\\[os,container *\\]");
+    }
```

Comment:
This test has no meaning in OpenJDK 8u. There, all container related printouts are guarded with `PrintContainerInfo` which is false for this test. It makes sense for JDKs with unified logging, as there a warning was being printed to stdout/err whether or not logging was otherwise turned on or not. It's conceivable that it would catch the assert, but that would also happen in one of the cpu/memory tests with additional cgroup fs mounts.

TLDR: I suggest to skip these test changes in the backport. All it does is adding to exec time, but would always be tautologically true (especially with the `os,container` match).